src/System.Text.Encoding.CodePages/tests/EncodingCodePages.cs

```diff
@@ -501,6 +506,8 @@ public static void TestDefaultEncodings()
             Assert.Contains(mappedEncoding, CrossplatformDefaultEncodings().Union(CodePageInfo().Select(i => Map((int)i[0], (string)i[1]))));
         }
 
+        static partial void ValidateSerializeDeserialize(Encoding e);
```

Comment:
The tests compile for both netstandard1.3 and netstandard1.7.  A partial method like this was the easiest way to include the additional tests; when compiling with netstandard1.3, this will be a nop, and when compiling with netstandard1.7, the other file will be included that provides an implementation of this partial method.